File: uikit/src/main/java/io/agora/chat/uikit/chat/model/EaseInputMenuStyle.java
package io.agora.chat.uikit.chat.model;

/**
 * Input menu includes: voice input, emoji input, text input and extended function input
 */
public enum EaseInputMenuStyle {
    /**
     * Includes all functions
     */
    All,
    /**
     * Includes all functions except voice input
     */
    DISABLE_VOICE,
    /**
     * Includes all functions except emoji input
     */
    DISABLE_EMOJICON,
    /**
     * Includes all functions except voice input and emoji input
     */
    DISABLE_VOICE_EMOJICON,
    /**
     * Includes text input only
     */
    ONLY_TEXT
}

File: ease-im-kit/src/main/kotlin/io/agora/uikit/feature/chat/widgets/EaseInputMenuStyle.kt
package io.agora.uikit.feature.chat.widgets

/**
 * Input menu includes: voice input, emoji input, text input and extended function input
 */
enum class EaseInputMenuStyle {
    /**
     * Includes all functions
     */
    All,

    /**
     * Includes all functions except voice input
     */
    DISABLE_VOICE,

    /**
     * Includes all functions except emoji input
     */
    DISABLE_EMOJICON,

    /**
     * Includes all functions except voice input and emoji input
     */
    DISABLE_VOICE_EMOJICON,

    /**
     * Includes text input only
     */
    ONLY_TEXT
}